src/main/java/com/celements/navigation/Navigation.java

```diff
@@ -413,19 +414,23 @@ void addNavigationForParent(StringBuilder outStream, EntityReference parentRef,
       } else if ((getCurrentLevel(numMoreLevels) == 1) && hasedit()) {
         LOGGER.trace("addNavigationForParent: empty navigation hint for parent [" + parentRef
             + "] numMoreLevels [" + numMoreLevels + "], currentLevel [" + getCurrentLevel(
-                numMoreLevels) + "].");
+                numMoreLevels)
```

Comment:
please leave it on one line. This line break does not make it more readable. Instead you could us the message.format to make it more organised.